chain/network/src/routing/routing_table_view.rs

```diff
@@ -12,24 +11,28 @@ use std::sync::Arc;
 use tracing::warn;
 
 const ANNOUNCE_ACCOUNT_CACHE_SIZE: usize = 10_000;
-const ROUND_ROBIN_MAX_NONCE_DIFFERENCE_ALLOWED: usize = 10;
-const ROUND_ROBIN_NONCE_CACHE_SIZE: usize = 10_000;
+const LAST_ROUTED_CACHE_SIZE: usize = 10_000;
+
+type RoutingTable = Arc<HashMap<PeerId, Vec<PeerId>>>;
 
 pub(crate) struct RoutingTableView {
-    /// PeerId associated for every known account id.
-    account_peers: LruCache<AccountId, AnnounceAccount>,
-    /// Active PeerId that are part of the shortest path to each PeerId.
-    pub(crate) peer_forwarding: Arc<HashMap<PeerId, Vec<PeerId>>>,
+    my_peer_id: PeerId,
     /// Store last update for known edges. This is limited to list of adjacent edges to `my_peer_id`.
-    pub(crate) local_edges_info: HashMap<PeerId, Edge>,
+    local_edges_info: HashMap<PeerId, Edge>,
+
+    /// Maps an account_id to a peer owning it.
+    account_peers: LruCache<AccountId, AnnounceAccount>,
+    /// Maps a peer_id to a set of connected peers that on the shortest path to peer_id.
+    routing_table: RoutingTable,
```

Comment:
Looking at this, perhaps

```rust
next_hop: NextHopTable
```

would be a better name? "RoutingTable" to me sounds like it should contain full routes, and the types makes it look like it does, but in reality only next hop is stored I think?

Reply:
(even if it is, def out of scope for this PR though!)

Reply:
done